Remove client-sideish settings

DerZade opened the pull request:
Remove forced settings, which shouldn't be forced because they only have a client-side effect.

### `ace_medical_feedback_bloodVolumeEffectType`
**Ace Medical > Low Blood Volume Effect Type**
Selects the used low blood volume effect type. (Options: `Color Fading`, `Icon`, `Icon + Color Fading`)

###  `ace_microdagr_mapDataAvailable`
**ACE Uncategorized > MicroDAGR Map Fill**
How much map data is filled on MicroDAGR's (Options: `None (Cannot use map view)`,`Topographical + Roads`, `Full Satellite + Buildings`)

### `ace_nametags_playerNamesMaxAlpha`
**ACE Name Tags > Player tags transparency**

### `ace_nametags_playerNamesViewDistance`
**ACE Name Tags > Player Names View Dist.**
Distance in meters at which player names are shown.

### `ace_nametags_showCursorTagForVehicles`
**ACE Name Tags > Show for Vehicles**
Show cursor Name Tag for vehicle commander (only if client has name tags enabled)

### `ace_viewdistance_enabled`
**ACE View Distance Limiter > Enable ACE viewdistance**
Enables ACE viewdistance

### `ace_viewdistance_limitViewDistance`
**ACE View Distance Limiter > View Distance Limit**
Limit for client's view distance set here and can be overridden by module.

### `acex_volume_enabled`
**ACEX Volume > Enabled**
Toggle volume reduction.

DerZade committed:
Update cba_settings.sqf

Fusselwurm commented on cba_settings.sqf:
```diff
@@ -199,8 +194,6 @@ force ace_ui_allowSelectiveUI = true;
 force ace_vehiclelock_lockVehicleInventory = true;
 force ace_vehiclelock_vehicleStartingLockState = -1;
-force ace_viewdistance_enabled = true;
-force ace_viewdistance_limitViewDistance = 10000;
```
wait, gabs nich irgendwelche fuckiness, daß aufm server die höchstmögliche view distance gesetzt sein muß, damit die clients nicht begrenzt werden? wenn ja, sollten wir nur das `force` rausnehmen

oder gings dabei nur um die profile-viewdistance und nix mit ace <-this probably

Fusselwurm replied:
@gruppe-adler/adler brauch hier mal gesammelte Erfahrungswerte

Fusselwurm replied:
ftr: der Gedanke lt. @AdlerSalbei damals war, dem DAU der zum ersten mal ACE in die Hand nimmt, eine vernünftige Einstellung zu geben.
 
Da aber hier: https://github.com/acemod/ACE3/blob/be09fd10a3e51cdd90bf8e3d2df6f000fdec2f5a/addons/viewdistance/ACE_Settings.hpp#L5 das per default an ist, können wir das setting tatsächlich rausschmeißen 

👍

DerZade replied:
Perfekt :)

Fusselwurm reviewed (approved):
hat alles Sinn, nur mit der viewdistance bin ich grad unsicher - aber wenn das läuft dann ab dafür :)

Fusselwurm commented on cba_settings.sqf:
```diff
@@ -92,7 +92,6 @@ force ace_medical_blood_bloodLifetime = 900;
 force ace_medical_blood_enabledFor = 2;
 force ace_medical_blood_maxBloodObjects = 500;
 force ace_medical_fatalDamageSource = 0;
-force ace_medical_feedback_bloodVolumeEffectType = 0;
```
👌 ist auch der default

Fusselwurm commented on cba_settings.sqf:
```diff
@@ -130,14 +129,10 @@ force ace_medical_treatment_medicIV = 1;
 force ace_medical_treatment_medicPAK = 1;
 force ace_medical_treatment_medicSurgicalKit = 1;
 force ace_medical_treatment_timeCoefficientPAK = 1;
-force ace_microdagr_mapDataAvailable = 2;
```
default 👌

Fusselwurm commented on cba_settings.sqf:
```diff
@@ -218,7 +211,6 @@ force acex_headless_enabled = false;
 force acex_headless_endMission = 0;
 force acex_headless_log = false;
 force acex_sitting_enable = true;
-force acex_volume_enabled = true;
```
da is der default false - um das bisherige Verhalten nicht zu ändern (außer für die die es wirklich wollen) würd ich hier lediglich das "force" rausnehmen wollen

Fusselwurm commented on cba_settings.sqf:
```diff
@@ -130,14 +129,10 @@ force ace_medical_treatment_medicIV = 1;
 force ace_mk6mortar_useAmmoHandling = false;
-force ace_nametags_playerNamesMaxAlpha = 0.8;
-force ace_nametags_playerNamesViewDistance = 5;
-force ace_nametags_showCursorTagForVehicles = false;
```
sind auch alles die defaults 👍

Fusselwurm committed:
switch default for acex_volume_enabled

acex_volume defaults to *disabled*, and in our context it should default to *enabled*

Fusselwurm commented:
2 1 risiko

Fusselwurm commented:
ah curses, 717b32c  funzt nich ganz so wie ich dachte. also... der client-_default_ zählt genauso schwer wie ein _explizites_ client-setting. 

d.h. server setting ohne force wird *nie* auf den client angewendet

:derp:
